changelogs/current.yaml

```diff
@@ -53,6 +53,9 @@ minor_behavior_changes:
 - area: cache_filter
   change: |
     added a completion callback to insertHeaders and insertTrailers in cache interface. Any external cache implementation extensions will need to also add this callback, and call it on completion.
+- area: local_ratelimit
+  change: |
+    local ratelimit support virtualhost config.
```

Comment:
nit: added :ref:`virtual host level configuration <envoy_v3_api_field_extensions_..........>` support for the local ratelimit filter.

Reply:
done, thanks.